src/openms/extern/SQLiteCpp/.travis.yml

```
@@ -34,84 +46,110 @@ jobs:
+    - dist: focal
+      env:
+        - cc=clang cxx=clang++
+
+    # Clang 7.0.0 (Ubuntu Bionic 18.04)
     - dist: bionic
       env:
         - cc=clang cxx=clang++
 
-    # Clang 7.0.0
-    - dist: xenial
+    # Clang 7.0.0 (Ubuntu Xenial 16.04) 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)